agent/agentssh/agentssh.go

```diff
@@ -206,25 +209,26 @@ func (s *Server) sessionHandler(session ssh.Session) {
 		s.sftpHandler(session)
 		return
 	default:
-		s.logger.Debug(ctx, "unsupported subsystem", slog.F("subsystem", ss))
+		logger.Warn(ctx, "unsupported subsystem", slog.F("subsystem", ss))
```

Comment:
This could just as well be info, or debug IMO. Not sure why we want to warn about this since the client is doing something fringe.

```suggestion
		logger.Warn(ctx, "unsupported subsystem, closing session", slog.F("subsystem", ss))
```

Reply:
client "doing something fringe" should be uncommon, and relatively likely to generate a support request.  I definitely want stuff like that to show up in our logs so we can easily tell that the problem is their client misbehaving and not a Coder bug.